l_x_v_front: &lxfv 0.76574
l_x_v_back: &lxbv -0.76574
l_x_h_front: &lxfh 1.22921
l_x_h_back: &lxbh -1.19239
l_y_v_left: &lylv 0.570
l_y_v_right: &lyrv -0.570
l_y_h_left: &lylh 0.56425
l_y_h_right: &lyrh -0.56425
l_z_v: &lzv 0.577
l_z_h: &lzh -0.054


thrusters:
  '0':
    f: [ 0.00, -0.50, 0.866]
    r: [ *lxbv, *lylv, *lzv]
    k_forward: 1.00
    k_backward: 1.00

  '1':
    f: [ 0.00, 0.50, 0.866]
    r: [ *lxbv, *lyrv, *lzv]
    k_forward: 1.00
    k_backward: 1.00

  '2':
    f: [ 0.00, -0.50, 0.866]
    r: [ *lxfv, *lylv, *lzv]
    k_forward: 1.00
    k_backward: 1.00

  '3':
    f: [ 0.00, 0.50, 0.866]
    r: [ *lxfv, *lyrv, *lzv]
    k_forward: 1.00
    k_backward: 1.00

  '4':
    f: [ 0.707, 0.707, 0.00]
    r: [ *lxbh, *lylh, *lzh]
    k_forward:  1.00
    k_backward:  1.00

  '5':
    f: [ 0.707, -0.707, 0.00]
    r: [ *lxbh, *lyrh, *lzh]
    k_forward:  1.00
    k_backward:  1.00

  '6':
    f: [ -0.707, 0.707, 0.00]
    r: [ *lxfh, *lylh, *lzh]
    k_forward:  1.00
    k_backward:  1.00

  '7':
    f: [ -0.707, -0.707, 0.00]
    r: [ *lxfh, *lyrh, *lzh]
    k_forward:  1.00
    k_backward:  1.00
